### DISCUSSION TITLE
Just to keep on conversation

### DISCUSSION THREAD

**Discussion by OlliSant** [a-cosmology-group/acg#236] [Metaphysics and Philosophy]
My newest update https://www.santavuori.com/Galakseja3a.pdf
The universe does not begin. It is eternal and boundless.

**Comment by OlliSant** [comment]
More insights. The size of the universe: more than 10 billion light years, and less than infinity.

**Reply by RedshiftDrift** [reply]
You can say "more than 35 billion light years", the new galaxy [JADES-GS-z14-0](https://en.wikipedia.org/wiki/List_of_the_most_distant_astronomical_objects#Most_distant_spectroscopically-confirmed_objects) recently discovered is at least that far!

**Reply by OlliSant** [reply]
Yes! The approximate size of the universe is between infinity and 35 billion light years. If ballike structure, the diameter 70 billion light years or more, but not absolute infinity. And no beginning.

**Reply by mikehelland** [reply]
For what it's worth, in my model, z=14 is at a distance of 12.9 Gly:

$$d = \frac{z}{1+z} \frac{c}{H}$$

Things aren't nearly as spread out in the model, which I think gives a much more sane account for the number density of galaxies and quasars at high redshift. IOW, a z=10 galaxy and a z=20 galaxy are actually pretty close to each other.

**Reply by OlliSant** [reply]
Say it verbally, how big is the universe, minimum.

**Reply by mikehelland** [reply]
The observable region of space has a radius of 13.8 billion light years in that model.

**Reply by RedshiftDrift** [reply]
JWST's "little red dots" look like redshifts in the $z > 28$ range.  That a minimum radius of 44 Gly for the observed universe.

**Reply by OlliSant** [reply]
88 Gly diameter! Big.

**Reply by OlliSant** [reply]
Only one universe, only one space for all. But many sub-universes. Eternal universe, dynamic sub-universes. 

AI made easily the mathematics for my verbal model! Just make a verbal model, and AI makes the mathematics and the good points and bad points and how to test. Just ask more and more.

So easy now. But my competence is not in physics.
